frontend/src/views/pages/DashboardPage.tsx

```diff
@@ -1,27 +1,36 @@
 import { Typography, Stack, Box } from "@mui/material";
 import { useLocation } from "react-router-dom";
-import Overview from "../components/Dashboard/Overview/Overview";
 import GraphGrid from "../components/Dashboard/GraphGrid/GraphGrid";
-import BiasScore from "../components/Dashboard/BiasScore/BiasScore";
+import Overview from "../components/Dashboard/Overview/Overview";
+import LetterGrade from "../components/Dashboard/LetterGrade/LetterGrade";
+import Footer from "../components/Footer/Footer";
 
 function DashboardPage() {
   const location = useLocation();
   const { dashboardData } = location.state;
 
   return (
+    <>
     <Stack>
-      <Typography variant="body2" pb={3}>
-        {dashboardData.filePath}
+      <Typography tabIndex={0} aria-label={`File path displayed as ${dashboardData.filePath.split('/').pop()}`} p={14} sx={{textAlign:'left', fontFamily: 'Montserrat', color: "#9921D2", fontWeight: 500, fontSize: '15pt'}} pb={3}>
+        {dashboardData.filePath.split('/').pop()}
       </Typography>
 
-      <Box bgcolor="#E6EEF5" p={10}>
-        <Stack direction="row" spacing={5}>
-          <BiasScore score={dashboardData.overview.score} />
-          <Overview data={dashboardData.overview} />
+      <Stack sx={{backgroundColor: '#E6EEF5'}}>
```

Comment:
We might want to add page structure, specifically page regions like <header> <main> and <aside>. I think it is important for accessibility for screen readers! https://www.w3.org/WAI/tutorials/page-structure/regions/

If there are page regions of the same type (I think this might be the case) we can use region labels: https://www.w3.org/WAI/tutorials/page-structure/labels/.

For future ref: https://www.w3.org/WAI/tutorials/page-structure/